chart/infra-server/static/workflow-openshift-4-demo.yaml

```diff
@@ -75,7 +76,7 @@ spec:
 
     - name: create
       container:
-        image: quay.io/stackrox-io/ci:automation-flavors-openshift-4-0.6.11
+        image: quay.io/stackrox-io/ci:automation-flavors-openshift-4-0.8.0-1-gd92b417d7f-snapshot
```

Comment:
Just a reminder for you to update the image after merging + tagging: https://github.com/stackrox/automation-flavors/pull/152 .